src/dds/participant.rs

```diff
@@ -53,7 +53,7 @@ pub struct DomainParticipant {
 impl DomainParticipant {
   /// # Examples
   /// ```
-  /// # use rustdds::DomainParticipant;
```

Comment:
Is there a reason we have the imports commented out some of the time? Seems counter intuitive as you should able to run these examples with rustdoc

Reply:
That syntax doesn't comment them out, just hides them from the output, for brevity